src/org/ses/android/soap/tasks/BuscarHuellaTask.java

```diff
@@ -16,16 +16,17 @@
 /**
  * Created by franciscorivera on 1/8/16.
  * Used to query the SOAP API for a fingerprint and retrieve the
- * participant object that the fingerprint belongs to (null if
+ * codigoPaciente that the fingerprint belongs to (null if
  * error)
  */
 
-public class BuscarHuellaTask extends AsyncTask<Byte, Void, Participant> {
+public class BuscarHuellaTask extends AsyncTask<String, String, String> {
     
     @Override
-    protected Participant doInBackground(Byte ... params)
+    protected String doInBackground(String ... params)
```

Comment:
Can you explain in a comment here which parameters are needed? (e.g. param 0 = fingerprint encoded as a base64 string)